<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTableV</class>
 <widget class="QMainWindow" name="QtTableV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTableV</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>TableView</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>PlainText</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionPreViewData"/>
   <addaction name="separator"/>
   <addaction name="actionAppRow"/>
   <addaction name="actionInsertRow"/>
   <addaction name="actionDeleteRow"/>
   <addaction name="separator"/>
   <addaction name="actionAlignLeft"/>
   <addaction name="actionAlignCenter"/>
   <addaction name="actionAlignRight"/>
   <addaction name="actionTextBold"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_table/dakaiwenjianjia.svg</normaloff>:/icon_table/dakaiwenjianjia.svg</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>OpenFile</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPreViewData">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_table/yulan.svg</normaloff>:/icon_table/yulan.svg</iconset>
   </property>
   <property name="text">
    <string>数据预览</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAppRow">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_table/tianjiahang.svg</normaloff>:/icon_table/tianjiahang.svg</iconset>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionInsertRow">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_table/light-component-kv-KVinsert.svg</normaloff>:/icon_table/light-component-kv-KVinsert.svg</iconset>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDeleteRow">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_table/shanchuhang.svg</normaloff>:/icon_table/shanchuhang.svg</iconset>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_table/juzuo.svg</normaloff>:/icon_table/juzuo.svg</iconset>
   </property>
   <property name="text">
    <string>居左</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAlignCenter">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_table/juzhong.svg</normaloff>:/icon_table/juzhong.svg</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_table/juyou.svg</normaloff>:/icon_table/juyou.svg</iconset>
   </property>
   <property name="text">
    <string>居右</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionTextBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_table/ziyuan.svg</normaloff>:/icon_table/ziyuan.svg</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_table/tuichu_2.svg</normaloff>:/icon_table/tuichu_2.svg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
